edge-modules/api-proxy-module/build/linux/test.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+###############################################################################
```

Comment:
Talked about this file.  Your intention is to keep similar as other test script in edgelet or mqtt broker, and you expect these script can be merged/combined into 1 file. 

Not a big deal to me, but based on my experience, this never happens.

Reply:
I am logging a PBI to talk about the tests.